django_tasks/backends/database/admin.py

```diff
@@ -0,0 +1,22 @@
+from typing import List, Optional
+
+from django.contrib import admin
+from django.http import HttpRequest
+
+from .models import DBTaskResult
+
+
+@admin.register(DBTaskResult)
+class DBTaskResultAdmin(admin.ModelAdmin):
```

Comment:
Suggestion: Now that we're exposing their names, it might be nice to set nicer names for the module and the `verbose_name` for the model itself, so the admin interface looks a little nicer.